.github/workflows/docs-deploy.yml

```diff
@@ -6,6 +6,7 @@ on:
     branches: ["master", "release/*"]
     paths:
       - ".github/workflows/docs-deploy.yml"
+      # FIXME: what is this job for?
```

Comment:
@Borda I'm confused about this workflow. Right now it's set to only run if this file is modified. Is this intended? If so, why?

Reply:
it was, because before it did not run for any PR so it was hard to debug, but may good idea is to extend the range of chnages

Reply:
Do you have any suggestions? Anyways, this can be done in a separate PR. I'll add a TODO